packages/components/drawer/src/index.ts

```diff
@@ -1,9 +1,26 @@
-import {ModalHeader, ModalBody, ModalFooter, ModalContent} from "@nextui-org/modal";
+import {
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  ModalContent,
+  ModalContentProps,
+  ModalHeaderProps,
+  ModalBodyProps,
+  ModalFooterProps,
+  UseDisclosureProps,
```

Comment:
remove the `UseDisclosureProps` export, it will have conflicts with the one exported from the `modal` package